neurots/generate/grower.py

```diff
@@ -82,14 +82,14 @@ def __init__(
         input_distributions,
         context=None,
         external_diametrizer=None,
-        skip_validation=False,
+        skip_preprocessing=False,
         rng_or_seed=np.random,
         trunk_orientations_class=OrientationManager,
     ):
         """Constructor of the NeuronGrower class."""
         self.neuron = Morphology()
         self.context = context
-        self.skip_validation = skip_validation
+        self.skip_validation = skip_preprocessing
```

Comment:
```suggestion
        self.skip_preprocessing = skip_preprocessing
```
Let's be consistent everywhere